docs/dev/Build.md

```diff
@@ -309,8 +309,7 @@ Run following commands:
 cd /tmp
 wget https://golang.org/dl/go1.15.2.linux-amd64.tar.gz
 tar xaf go1.15.2.linux-amd64.tar.gz
-mv go go-1.15
```

Comment:
Please leave this as is. 
This works more correctly (i.e. actually it will not let do it, by produce error like `mv: cannot move 'yyy' to 'xxx/yyy': Directory not empty`) if you accidentally already have `/usr/local/go-xxx`.